taxonomies/food/ingredients.txt

```diff
@@ -5916,27 +5822,18 @@ wikipedia:en: https://en.wikipedia.org/wiki/Smetana_(dairy_product)
 
 < en:sour cream
 en: Crème fraîche
-af: Crème fraîche
+xx: crème fraîche
```

Comment:
Could probably include unaccented version too, to clean up the list even further.
```suggestion
xx: crème fraîche, creme fraiche
```

Reply:
I removed this one, for now